launch/tier4_localization_launch/launch/eagleye/fix2pose.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<launch>
+  <arg name="output_pose_with_cov_name" default="/localization/pose_estimator/pose_with_covariance"/>
+  <node pkg="eagleye_fix2pose" name="fix2pose_node" exec="fix2pose">
+    <!-- /eagleye/fix or  /eagleye/rtk_fix(rtk_dead_reckoning mode) or /GNSS/fix(GNSS pose only mode)-->
+    <remap from="eagleye/fix" to="eagleye/fix" />
+    <remap from="ekf_trigger_node" to="/localization/pose_twist_fusion_filter/trigger_node"/>
```

Comment:
Why use the pose_twist_fusion_filter's trigger?

Reply:
This is because ekf will not start without eagleye submitting its services.

Reply:
I see.
So you mean, the fix2pose_node invokes ekf_localizer instead of pose_initializer.
https://github.com/MapIV/eagleye/blob/29bcc17adce86af1ea4db5739582be0677c12c86/eagleye_util/fix2pose/src/fix2pose.cpp#L208-L219

Reply:
That's right.

Reply:
@YamatoAndo 
I have made changes to pose_initilizer and finished changing this PR accordingly. Could you please check again?

PR for pose_initilizer
https://github.com/autowarefoundation/autoware.universe/pull/2904